src/index.js

```diff
@@ -32,7 +32,7 @@ async function doScript() {
         ['require', '--dev', `drupal/core-dev:${drupalVersion}`],
     ];
     if (utils.getMajorVersionFromConstraint(drupalVersion) > 8) {
-        commands.push(['require', '--dev', '--with-all-dependencies', 'phpspec/prophecy-phpunit:^2']);
+        commands.push(['require', '--dev', '--update-with-all-dependencies', 'phpspec/prophecy-phpunit:^2']);
```

Comment:
Per https://getcomposer.org/doc/03-cli.md, the option is `--with-all-dependencies`. What version of Composer is giving this issue?